src/robot_interface/telemetry/payloads.py

```diff
@@ -82,3 +78,38 @@ class RobotHeartbeatPayload:
     isar_id: str
     robot_name: str
     timestamp: datetime
+
+
+@dataclass
+class MissionPayload:
+    isar_id: str
+    robot_name: str
+    mission_id: Optional[str]
+    status: Optional[MissionStatus]
+    error_reason: Optional[ErrorReason]
+    error_description: Optional[str]
+    timestamp: datetime
+
+
+@dataclass
+class TaskPayload:
+    isar_id: str
+    robot_name: str
+    mission_id: Optional[str]
+    task_id: Optional[str]
+    status: Optional[TaskStatus]
+    task_type: Optional[TaskTypes]
+    error_reason: Optional[ErrorReason]
+    error_description: Optional[str]
+    timestamp: datetime
+
+
+@dataclass
+class InspectionResultPayload:
+    isar_id: str
+    robot_name: str
+    inspection_id: str
+    inspection_path: Union[str, dict]
+    installation_code: str
+    analysis_to_be_run: Optional[str]
```

Comment:
Does this have to be optional?

Reply:
Yes. Its being used the send analysis_type that is defined like: 
`analysis_type: Optional[str] = field(default=None, init=False)`